Samples/Client/WebClient/index.html

```diff
@@ -107,6 +176,11 @@
             pc.onopen = onSessionOpened;
             pc.onaddstream = onRemoteStreamAdded;
             pc.onremovestream = onRemoteStreamRemoved;
+			pc.ondatachannel = function(ev) {
```

Comment:
nit: spacing looks off here

Reply:
hmm.. have been using VisualStudioCode as raw html ediitor .  it is apparently mixing spaces and tabs.. will update to use spaces for consistency